### DISCUSSION TITLE
Feature request: Mapbox Search UI

### DISCUSSION THREAD

**Discussion by olofho** [rnmapbox/maps#1073] [Ideas]
## Mapbox search UI

Mapbox has a search function (https://www.mapbox.com/place-search/) that enables users to search for many different things, for example searching for restaurants would return a list of nearby restaurants. They have also created UI-components for this in both android and iOS. 

If possible, being able to use these components in react native would be a great feature!

**Comment by ClayC-WeatherBug** [comment]
I think that this is a great idea, but I wonder if this should be in a new repository in the interest of keeping the maps package smaller. Maybe we should do a new repository under the react-native-mapbox-gl organization for this package?
